src/app/shared/consts.ts

```diff
@@ -21,16 +22,20 @@ export const PREFIXES: Prefix[] = [
 ];
 
 export const LANGUAGES: Option[] = [
-  { code: LanguageCode.pl_PL, label: LanguageNames.POLISH },
-  { code: LanguageCode.uk_UA, label: LanguageNames.UKRAINIAN },
-  { code: LanguageCode.en_GB, label: LanguageNames.ENGLISH },
-  { code: LanguageCode.ru_RU, label: LanguageNames.RUSSIAN },
+  { code: AccommodationOffer.HostLanguageEnum.Pl, label: LanguageNames.POLISH },
+  { code: AccommodationOffer.HostLanguageEnum.Ua, label: LanguageNames.UKRAINIAN },
 ];
 
 export const LENGTHOFSTAY: Option[] = [
-  { code: '1t', label: '1 tydzień ' },
-  { code: '2t', label: '2 tygodnie' },
-  { code: '1m', label: '1 miesiąc ' },
-  { code: '2m', label: '2 miesiące' },
-  { code: 'o', label: 'dłużej' },
+  { code: AccommodationOffer.LengthOfStayEnum.Week1, label: '1 tydzień ' },
+  { code: AccommodationOffer.LengthOfStayEnum.Week2, label: '2 tygodnie' },
+  { code: AccommodationOffer.LengthOfStayEnum.Month1, label: '1 miesiąc ' },
+  { code: AccommodationOffer.LengthOfStayEnum.Month2, label: '2 miesiące' },
```

Comment:
I think in dto us also 3 months. Probably you need to regenerate them. By the way. In My PR this work is duplicated